syncano_cli/sync/project.py

```diff
@@ -48,13 +48,34 @@ def update_from_instance(self, instance, all=False, classes=None,
                              scripts=None):
         """Updates project data from instances"""
         LOG.info("Pulling instance data from syncano")
+        classesI = self.classes
```

Comment:
classesI doesn't tell anything about this variable. Use prev_classes or saved_classess.